site/docs/v1/tech/apis/_family-pending-response-body.adoc

```diff
@@ -40,16 +40,58 @@ The Id of the Tenant that this User belongs to.
 [field]#users``[x]``.timezone# [type]#[String]#::
 The User's preferred timezone. This can be used as a default to display instants, and it is recommended that you allow User's to change this per-session. The string will be in an https://www.iana.org/time-zones[IANA] time zone format.
 
-[field]#users``[x]``.twoFactorDelivery# [type]#[String]#::
+[field]#users``[x]``.twoFactor.methods``[x]``.authenticator.algorithm# [type]#[TOTPAlgorithm]#::
+The algorithm used by the TOTP authenticator. With the current implementation, this will always be `HmacSHA1`.
+
+[field]#users``[x]``.twoFactor.methods``[x]``.authenticator.codeLength# [type]#[Integer]#::
+The length of code generated by the TOTP. With the current implementation, this will always be `6`.
+
+[field]#users``[x]``.twoFactor.methods``[x]``.authenticator.timeStep# [type]#[Integer]#::
+The time-step size in seconds. With the current implementation, this will always be 30.
+
+[field]#users``[x]``.twoFactor.methods``[x]``.email# [type]#[String]#::
+The value of the email address for this method. Only present if [field]#user.twoFactor.methods``[x]``.method# is `email`.
+
+[field]#users``[x]``.twoFactor.methods``[x]``.id# [type]#[UUID]#::
```

Comment:
This is not a UUID. This is a string. Either that or `methodId` is incorrect here: https://fusionauth.io/docs/v1/tech/apis/two-factor/#disable-multi-factor